src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -615,12 +615,13 @@ public final class JavadocTokenTypes {
      * <pre><code>{&#64;literal #compare(Object)}</code></pre>
      * <b>Tree:</b>
      * <pre>
-     * <code> |--JAVADOC_INLINE_TAG[1x0] : [{&#64;literal #compare(Object)}]
-     *        |--JAVADOC_INLINE_TAG_START[1x0] : [{]
-     *        |--LITERAL_LITERAL[1x1] : [@literal]
-     *        |--WS[1x9] : [ ]
-     *        |--TEXT[1x10] : [#compare(Object)]
-     *        |--JAVADOC_INLINE_TAG_END[1x27] : [}]
+     * <code>
+     *     |--JAVADOC_INLINE_TAG -&gt; JAVADOC_INLINE_TAG [5:27]
+     *        |--JAVADOC_INLINE_TAG_START -&gt; { [5:27]
+     *        |--LITERAL_LITERAL -&gt; @literal [5:28]
+     *        |--WS -&gt;   [5:36]
```

Comment:
Please remove line and column numbers.
From all lines.

Reply:
done